@here/olp-sdk-dataservice-read/test/unit/CatalogClient.test.ts

```diff
@@ -132,6 +134,33 @@ describe("CatalogClient", () => {
         assert.isTrue(response.versions.length > 0);
     });
 
+    it("Should method getLayerVersions provide data with version parameter", async () => {
+        const mockedVersion = {
+            layerVersions: [
+                { layer: "testLayer1", version: 1, timestamp: 11 },
+                { layer: "testLayer2", version: 2, timestamp: 22 },
+                { layer: "testLayer3", version: 3, timestamp: 33 }
+            ],
+            version: 3
+        };
+
+        getLayerVersionsStub.callsFake(
+            (builder: any, params: any): Promise<MetadataApi.LayerVersions> => {
+                return Promise.resolve(mockedVersion);
+            }
+        );
+
+        const catalogRequest = new dataServiceRead.LayerVersionsRequest().withVersion(
```

Comment:
Please add test case with calling getLayerVersions without set version in the LayerVersionsRequest